tcms/testruns/templates/testruns/get.html

```diff
@@ -413,6 +414,10 @@ <h5><span class="test-executions-count"></span> Results</h5>
     </div> <!-- /row -->
 </div>
 
+<template id="status-button-template">
+    <i style="font-size: 28px; margin-right: 10px; cursor: pointer;"></i>
+</template>
```

Comment:
This is passed to the template as context variable `execution_statuses` and can be rendered by Django, no need to render it ourselves.